src/tools/pkcs11-tool.c

```diff
@@ -5576,18 +5579,18 @@ static int read_object(CK_SESSION_HANDLE session)
 			EC_KEY_set_public_key(EVP_PKEY_get0_EC_KEY(pkey), point);
 #else
 			if (!(bld = OSSL_PARAM_BLD_new()) ||
-				EVP_PKEY_todata(pkey, EVP_PKEY_PUBLIC_KEY, &old) != 1 ||
-				OSSL_PARAM_BLD_push_octet_string(bld, "pub", a, a_len) != 1 ||
-				!(new = OSSL_PARAM_BLD_to_param(bld)) ||
-				!(p = OSSL_PARAM_merge(old, new))) {
-					OSSL_PARAM_BLD_free(bld);
-					OSSL_PARAM_free(old);
-					OSSL_PARAM_free(new);
-					OSSL_PARAM_free(p);
-					if (success)
-						ASN1_STRING_free(os);
-					free(value);
-					util_fatal("cannot set OSSL_PARAM");
+			    EVP_PKEY_todata(pkey, EVP_PKEY_PUBLIC_KEY, &old) != 1 ||
+			    OSSL_PARAM_BLD_push_octet_string(bld, "pub", a, a_len) != 1 ||
+			    !(new = OSSL_PARAM_BLD_to_param(bld)) ||
+			    !(p = OSSL_PARAM_merge(old, new))) {
+				OSSL_PARAM_BLD_free(bld);
```

Comment:
Here you have an example where the formatting actually makes the code harder to read... The block encapsulated in `{}` is suddenly aligned identically as the conditions from the `if` clauses. So let's don't mix tabs/spaces to align the breaks with the opening brackets as argued above

Reply:
This is true for one configuration where you have tab width=4 (because the length of "if (" is 4). For all the other cases, the block is aligned differently than the condition.